userspace/falco/falco.cpp

```diff
@@ -624,6 +624,7 @@ int falco_init(int argc, char **argv)
 				if(string(long_options[long_index].name) == "version")
 				{
 					printf("Falco version: %s\n", FALCO_VERSION);
+					printf("Driver version: %s\n", DRIVER_VERSION);
```

Comment:
So we don't need to answer this now - but we should start thinking about what/when/where we output various pieces of information in Falco

We could standardize some of this - and I think we should give meaningful information whenever possible

Reply:
I agree.

In this case the reasoning was: Falco version is a different thing than the driver version now. Thus, we need to communicate it to the users. :)